ui/README.md

```diff
@@ -1,2 +1,37 @@
-# Anbox Cloud - Cloud Gaming Demo
+# Web UI for cloud gaming demo
 
+The Web UI for clouding gaming demo is built with [flutter](https://flutter.dev/).
+
+## Requirements
+
+To setup Web development and build the UI, make sure that you have flutter installed.
+Please follow the [instructions](https://docs.flutter.dev/get-started/install/linux) to install the flutter on your machine.
+
+## Build
+
+To build the UI, simply run:
+
+    ```
+    flutter build web
+    ```
+
+This generates assets, js, html files that are required to run the Web UI and places all of them into the `build/web` folder.
+Please refer to the official [documentation](https://docs.flutter.dev/get-started/web) for more details on how to build Web apps with flutter.
+
+## Deploy
+
+Create a `static` folder under the backend service and copy everything that is in the `build/web` folder to the `static` folder.
+
+    ```
+    mkdir -p backend/service/static
+    cp av build/web/* backend/service/static
+    ```
+
+This enables Web UI to be hooked up with the backend service.
```

Comment:
```suggestion
This allows the UI to be executed together with the backend service.

Reply:
How about:
```suggestion
The backend service will then use this version of the web UI.
```
Not sure I understand exactly how this works though. My guess is that if you don't build the web UI yourself, it will be built by the installation script. And if there is anything in the `static` folder, that version is used instead. Is that correct?